pkgs/node/src/node.zig

```diff
@@ -239,7 +395,7 @@ pub const BeamNode = struct {
                                 self.node_registry.getNodeNameFromPeerId(block_ctx.peer_id),
                             });
 
-                            self.processBlockByRootChunk(block_ctx, &block_resp);
+                            try self.processBlockByRootChunk(block_ctx, &block_resp);
```

Comment:
what are the failure cases here?

Reply:
It doesn't return an error in most of the cases and just logs it expect following cases:
1. allocation errors
2. ssz clone errors
3. cache map put errors
4. fetchBlockByRoots errors